### DISCUSSION TITLE
And I've made a frakenstein mouthpiece

### DISCUSSION THREAD

**Discussion by C1R3Z5** [cardonabits/haxo-hw#65] [Show and tell]
If only I knew how to design a mouthpiece... I've cut and pasted two 3D printed mouthpieces together to make a suitable mouthpiece. One is a tenor sax mouthpiece and the other a Yamaha WX5 (it doesn't need a reed). It fits tight and works well!
![1000009876](https://github.com/cardonabits/haxo-hw/assets/155772303/e0129c6b-2f29-4f84-aa43-3791ff2958d9)

**Comment by C1R3Z5** [comment]
![20240105_143821](https://github.com/cardonabits/haxo-hw/assets/155772303/0e0c3971-2acb-431e-85cb-2c7e63afd350)

**Reply by jcard0na** [reply]
Nice 💚!  Green is the perfect color for a Frankenstein mouthpiece!
If you want to share it with others, I'll be happy to accept a pull request.  Just put it in the [models](https://github.com/cardonabits/haxo-hw/tree/main/models) directory.

**Comment by C1R3Z5** [comment]
Thanks, Mr. Cardona! Shortly after I posted that hacked mouthpiece someone posted exactly what I was looking for. Unfortunately i am unable to add the model to the directory. 
https://www.thingiverse.com/thing:6409825

**Reply by jcard0na** [reply]
That's fine!  Thanks for the link, that is useful already.
